### DISCUSSION TITLE
Joining Slack channel

### DISCUSSION THREAD

**Discussion by niparx** [cross-language-cpp/djinni-generator#153] [General]
Link from [main page](https://djinni.xlcpp.dev/) for joining [Slack channel](https://join.slack.com/t/mobilecpp/shared_invite/zt-1iofql6w0-lTeGqoFrzOT_lGu~rG1MUg) is not working anymore. I'm getting this:
`This link is no longer active. To join this workspace, you’ll need to ask the person who originally invited you for a new link.`

Can somebody renew the link?

**Comment by a4z** [comment]
Thank you, for security reasons, the link expires from time to time, even if it should not
I opened a PR with a new link: https://github.com/cross-language-cpp/cross-language-cpp.github.io/pull/30

The slack channel is not used much anymore. It has been a while since there was some conversation.
For djinni-related topics, GitHub discussions might be a better place.
